pkg/kubelet/cm/memorymanager/memory_manager.go

```diff
@@ -153,7 +153,7 @@ func NewManager(policyName string, machineInfo *cadvisorapi.MachineInfo, nodeAll
 
 // Start starts the memory manager under the kubelet and calls policy start
 func (m *manager) Start(activePods ActivePodsFunc, sourcesReady config.SourcesReady, podStatusProvider status.PodStatusProvider, containerRuntime runtimeService, initialContainers containermap.ContainerMap) error {
-	klog.Infof("[memorymanager] starting with %s policy", m.policy.Name())
+	klog.InfoS("[memorymanager] starting with policy", "policy", m.policy.Name())
```

Comment:
```suggestion
	klog.InfoS("Starting memorymanager", "policy", m.policy.Name())
```

Please remove all the memorymanager log prefixes